TwitchLib.Api.Helix/Webhooks.cs

```diff
@@ -14,6 +15,30 @@ public Webhooks(IApiSettings settings, IRateLimiter rateLimiter, IHttpCallHandle
         {
         }
 
+        #region GetWebhookSubscriptions
+        public Task<GetWebhookSubscriptionsResponse> GetWebhookSubscriptionsAsync(string after = null, int first = 20, string accessToken = null)
+        {
+            if (first < 1 || first > 100)
+                throw new BadParameterException("'first' must between 1 (inclusive) and 100 (inclusive).");
```

Comment:
Use nameof(first) instead of 'first'.
Use the constructor which has a parameter name, and pass along nameof(first). --> BadParameterException("SomeDescription", nameof(first))

Gives clearer feedback and is more refactor proof.